src/infrastructure/access/api.js

```diff
@@ -1,15 +1,14 @@
 const config = require('./../config');
 
-const rp = require('login.dfe.request-promise-retry');
+const { fetchApi } = require('login.dfe.async-retry');
 const jwtStrategy = require('login.dfe.jwt-strategies');
 
 const callApi = async (method, endpoint, correlationId, body) => {
   const token = await jwtStrategy(config.access.service).getBearerToken();
 
   try {
-    return await rp({
+    return await fetchApi(`${config.access.service.url}/${endpoint}`, {
```

Comment:
`json: true` isn't needed in the options of `fetchApi` anymore since it's always JSON.

Reply:
Updated